<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>490</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>300</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(11, 12, 30);</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>120</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.999464, fy:0.494006, stop:0.482955 rgba(14, 38, 74, 124), stop:0.806818 rgba(165, 165, 165, 0));

border:1px, rgba(255, 255, 255, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>120</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.0449186, fy:0.5, stop:0.482955 rgba(72, 57, 38, 124), stop:0.806818 rgba(165, 165, 165, 0));

border:1px, rgba(255, 255, 255, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="l_temp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>230</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>56</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 180);
font: 75 56pt &quot;Calibri&quot;;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>-29 C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_temp_9">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>100</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>14</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 180);
font: 9pt &quot;Calibri&quot;;
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>швидкість вітру</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_wind">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>447</y>
     <width>90</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>14</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 180);
font: 9pt &quot;Calibri&quot;;
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>3 м/с</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_feel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>447</y>
     <width>90</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>14</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 180);
font: 9pt &quot;Calibri&quot;;
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>0 C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_hum_text">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>90</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>14</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 180);
font: 9pt &quot;Calibri&quot;;
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>вологість</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_temp_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>90</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>14</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 180);
font: 9pt &quot;Calibri&quot;;
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>відчувається</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_hum">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>447</y>
     <width>90</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>14</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 180);
font: 9pt &quot;Calibri&quot;;
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>750</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_type">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 180);
font: 16pt &quot;Calibri&quot;;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>type weather</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_day">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 180);
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>Понедельник</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>100</horstretch>
     <verstretch>100</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="1.qrc">:/newPrefix/img/01d.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_more">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>185</y>
     <width>30</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgba(54, 255, 114, 80);
	background-color: rgba(255, 255, 255, 15);
	border: 1px solid;
	border-top-left-radius: 30px;
	border-bottom-left-radius: 30px;
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 50);
}</string>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="f_more">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>300</width>
     <height>460</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>460</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>460</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="box">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
   </layout>
  </widget>
  <widget class="QFrame" name="f_panel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(79, 79, 79,200);
border-top-left-radius: 20px;
border-top-right-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="l_time">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>2</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px;
background-color: rgba(255, 255, 255, 0);
color: rgb(54, 255, 114);</string>
    </property>
    <property name="text">
     <string>13:12:26</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>5</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 7px;
    background-color: rgb(231, 76, 60);
}
QPushButton:hover:hover{
    background-color: rgb(192, 57, 43);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="f_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>25</y>
     <width>300</width>
     <height>485</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>485</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>485</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(11, 12, 30);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="box2"/>
  </widget>
  <widget class="QFrame" name="f_footer">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>410</y>
     <width>290</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>290</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>290</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 15);
border: 1px solid;
border-radius: 25px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="l_feeling">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>5</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/1/Temperature.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_humidity">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>5</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/1/Water.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_speed">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>5</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/1/Wind.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="l_city">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 180);
font: 26pt &quot;Calibri&quot;;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="locale">
    <locale language="Ukrainian" country="Ukraine"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>City</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>f_background</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>l_temp</zorder>
  <zorder>l_temp_9</zorder>
  <zorder>l_wind</zorder>
  <zorder>l_feel</zorder>
  <zorder>l_hum_text</zorder>
  <zorder>l_temp_5</zorder>
  <zorder>l_hum</zorder>
  <zorder>l_type</zorder>
  <zorder>l_day</zorder>
  <zorder>f_more</zorder>
  <zorder>f_panel</zorder>
  <zorder>l_logo</zorder>
  <zorder>f_footer</zorder>
  <zorder>l_city</zorder>
  <zorder>btn_more</zorder>
 </widget>
 <resources>
  <include location="1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
